### PR TITLE
Bulk Actions for newly introduced Notifications

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**
[SQL Script for testing](https://github.com/corona-school/dev-db/blob/main/dev/SimulateMatche.pgsql)

Seems to work quite fine, I'll bulk test on DEV with some thousand matches.

### CODE REVIEW

File: common/notification/index.ts [hunk 1]
```diff
@@ -241,3 +257,56 @@ async function deliverNotification(concreteNotification: ConcreteNotification, n
     }
 }
 
+/* When introducing new notifications, it might sometimes make sense to schedule them "in retrospective" once for all existing users,
+   as if the user took that action at actionDate */
+export async function actionTakenAt(actionDate: Date, user: Person, actionId: string, notificationContext: NotificationContext, apply: boolean) {
```
Comment (lsc0498) [inline common/notification/index.ts:262]: Are we sure that we will never send mails to deactivated users?

Comment (Jonasdoubleyou) [reply common/notification/index.ts:262]: Very true, that check was missing. Good catch!

Comment (Jonasdoubleyou) [reply common/notification/index.ts:262]: Though at least in theory all matches of deactivated users should be dissolved (just noticed when trying to test this)

**Review by lsc0498 — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**